can-control.js

```diff
@@ -285,14 +284,14 @@ var Control = Construct.extend(
 
 			if(!element) {
 				dev.warn('can/control/control.js: Creating an instance of a named control without passing an element');
-
-				element = vDom().createElement('div');
+				
+				throw 'Creating an instance of a named control without passing an element';
```

Comment:
I think you get more consistent messaging / stack traces across browsers if you use `throw new Error('...')`. Also you don't need the dev warning if you're also throwing an error.